src/lib/core.js

```diff
@@ -4824,6 +4825,8 @@
 
 			if (target == null) {
 				prop = "result";
+			} else if (target.type == "idRef") {
```

Comment:
We might want to support appending to class refs or query refs too. So, instead of having all these special cases, how about evaluating the target as an expression (put it in the `args`)? We could then decide how to append to it based on the type at runtime. 

Besides, the special casing of `"symbol"` means this command doesn't support the new variable scopes (local/element/global).

Reply:
It's a good point.  @benpate do you have time to do a deeper change here?

Reply:
Corrected?